### ISSUE TITLE
Crash when installing Charmonium in 1.18.2 (https://modrinth.com/mod/charmonium)

### ISSUE THREAD

**Issue by NoEnd-yt** [jakobkmar/pacmc#33]
Stacktrace :
```
                    Failed to deserialize the response to the following request: https://api.modrinth.com/api/v1/mod/charmonium/version
                    Error message: Unexpected JSON token at offset 839: Unexpected 'null' value instead of string literal
JSON input: .....endencies":[{"version_id":null,"project_id":"pOQTcQmj","depe.....
                    Please open an issue with the above error message at https://github.com/jakobkmar/pacmc/issues/new
                    
                    
                    Failed to deserialize the response to the following request: https://api.modrinth.com/api/v1/mod/bpii4Xsa/version
                    Error message: Unexpected JSON token at offset 839: Unexpected 'null' value instead of string literal
JSON input: .....endencies":[{"version_id":null,"project_id":"pOQTcQmj","depe.....
                    Please open an issue with the above error message at https://github.com/jakobkmar/pacmc/issues/new
```
`pacmc` crashes when trying to install the [Charmonium](https://modrinth.com/mod/charmonium) mod in 1.18.2.

**Comment by jakobkmar** [comment]
Hey, thanks for the report, this is the same as #32 - basically Modrinth changed their dependency system and therefore broke the v1 API as well. I am sorry for the inconvenience and hope to release a new version which does not have this issue in the forseeable future.

**Comment by jakobkmar** [comment]
Should be fixed in [0.5.0](https://github.com/jakobkmar/pacmc/releases/tag/0.5.0).
